src/drivers/mct/main/cime_comp_mod.F90

```diff
@@ -532,7 +532,7 @@ module cime_comp_mod
        'Sa_u:Sa_v'
 
   character(CL) :: hist_a2x3hr_flds  = &
-       'Sa_z:Sa_topo:Sa_u:Sa_v:Sa_tbot:Sa_ptem:Sa_shum:Sa_dens:Sa_pbot:Sa_pslv:Faxa_lwdn:&
+       'Sa_z:Sa_topo:Sa_u:Sa_v:Sa_tbot:Sa_ptem:Sa_shum:Sa_dens:Sa_uovern:Sa_pbot:Sa_pslv:Faxa_lwdn:&
```

Comment:
CESM won't want this field. It should only be added conditionally.

Reply:
I have reversed the change because I realized this can be done from the land model. I have updated the file in my latest commit accordingly. Thanks!